build/Targets/References.targets

```diff
@@ -1,25 +1,6 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
   <Choose>
```

Comment:
I think I'll approach this slightly different; I'll see if I can make use of the existing extension points in MSBuld (Directory.Build.props) next and drop the file next to the Solution, and then include it in the solution.